### PR TITLE
Add test for aspect-ratio precision

### PR DESCRIPTION
**Pull request by rwlbuis**
Add test for aspect-ratio precision verifying all
calculations are correct for a fractional aspect ratio.

### CODE REVIEW

File: css/css-sizing/aspect-ratio/block-aspect-ratio-036.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<title>CSS aspect-ratio: div block size aspect ratio precision</title>
+<link rel="author" title="Google LLC" href="https://www.google.com/">
```
Comment (mrego) [inline css/css-sizing/aspect-ratio/block-aspect-ratio-036.html:3]: Mmm, this seems a copy&paste mistake.

File: css/css-sizing/aspect-ratio/block-aspect-ratio-036.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+<link rel="help" href="https://drafts.csswg.org/css-sizing-4/#aspect-ratio">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht" />
```
Comment (mrego) [inline css/css-sizing/aspect-ratio/block-aspect-ratio-036.html:5]: Could you add a "<meta assert" explaining what the test is checking exactly? Thanks.

**Review by mrego — APPROVED** [review]
LGTM, some minor comments inline.

File: css/css-sizing/aspect-ratio/block-aspect-ratio-036.html (hunk 1 above)
Comment (rwlbuis) [reply css/css-sizing/aspect-ratio/block-aspect-ratio-036.html]: Done.

File: css/css-sizing/aspect-ratio/block-aspect-ratio-036.html (hunk 2 above)
Comment (rwlbuis) [reply css/css-sizing/aspect-ratio/block-aspect-ratio-036.html:5]: Done.

**Pull request merged**